lcd/3Wire-HT1621B-GDC03849/gdc03849.py

```diff
@@ -0,0 +1,182 @@
+# -*- coding:UTF-8 -*-
+
+'''
+MIT License
+
+Copyright (c) 2018 Robin Chen
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+'''
+
+'''/******************************************************************************
+* File       : gdc03849.py
+* Function   : LCD Drive
+* description: To be done.
+* Version    : V1.00
+* Author     : Robin Chen
+* Date       : 9th May 2018
+* History    :  No.  When           Who           What
+*               1    9/May/2018     Robin Chen    Create
+*               2    30/May/2018    Robin Chen    Fengzhuang
+******************************************************************************/'''
+
+class GDC03849:
+    '''/******************************************************************************
+    * Name       : __init__(self, _ht1621x)
+    * Function   : Init
+    * Input      : _ht1621x: object.
+    * Output:    : None.
+    * Return     : None.
+    * description: To be done.
```

Comment:
I am expecting more description
by the way, is it "Description" or "description"?